applications/adv_networking_bench/adv_networking_bench_rdma_tx_rx.yaml

```diff
@@ -0,0 +1,126 @@
+%YAML 1.2
+# SPDX-FileCopyrightText: Copyright (c) 2022-2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+multithreaded: true
+num_delay_ops: 32
+delay: 0.1
+delay_step: 0.01
+
+scheduler:
+  check_recession_period_ms: 0
+  worker_thread_number: 5
+  stop_on_deadlock: true
+  stop_on_deadlock_timeout: 500
+  # max_duration_ms: 20000
+
+advanced_network:
+  cfg:
+    version: 1
+    manager: "rdma"
+    master_core: 3
+    debug: false
+    log_level: "info"
+
+    memory_regions:
+    - name: "DATA_RX_CPU_SERVER"
+      kind: "device"
+      affinity: 0
+      access:
+        - local
+      num_bufs: 20
+      buf_size: 9000000  
+    - name: "DATA_TX_CPU_SERVER"
+      kind: "device"
+      affinity: 0
+      access:
+        - local
+      num_bufs: 20
+      buf_size: 9000000        
+    - name: "DATA_TX_CPU_CLIENT"
+      kind: "device"
+      affinity: 0
+      access:
+        - local
+      num_bufs: 20
+      buf_size: 90000000       
+    - name: "DATA_RX_CPU_CLIENT"
+      kind: "device"
+      affinity: 0
+      access:
+        - local
+      num_bufs: 20
+      buf_size: 90000000           
+
+    interfaces:
+    - name: data1
+      rdma_mode: client
+      rdma_transport_mode: RC
+      address: 192.168.11.2       # The address to use, or leave blank for auto-detect
```

Comment:
This is the same field that we use for the NIC BUS address for other backends. Will IP be supported in other backends as well?
- If yes, great
- If not, we should use a separate field name than `address`

Reply:
It was intended to be an overloaded field so we didn't have to create multiple fields. "address" can be IP address, PCIe address, or interface name. It won't necessarily apply to all backends for all variants, but it gives more flexibility and I don't think there's ambiguity.

Reply:
follow up in https://github.com/nvidia-holoscan/holohub/pull/819#discussion_r2140484627